pubs/p3.py

```diff
@@ -52,6 +54,7 @@ def _get_fake_stdio_ucontent(stdio):
         return stdio.read()
 
     import pickle
+    from pipes import quote as shell_quote
```

Comment:
I don't get why this line is repeated and not put outside of the if/else if it is the same for both versions.

Reply:
Is it relevant in p3.py by the way?

Reply:
Well spotted! The py3 version of the line should be importing `quote` from `shlex`…